packages/web/src/scss/components/TextField/README.md

```diff
@@ -132,9 +132,28 @@ forget to change not only the input type but also `aria-pressed` and
 ## Validation States
 
 Validation states can be presented either by adding a CSS modifier class
-(`TextField--success`, `TextField--warning`, `TextField--error`), or by adding
+(`TextField--success`, `TextField--warning`, `TextField--danger`, `TextField--error`), or by adding
 a JS interaction class when controlled by JavaScript (`has-success`,
-`has-warning`, `has-error`).
+`has-warning`, `has-danger`, `has-error`). See Validation state [dictionary][docs].
```

Comment:
I would prefer better name for the dictionary [docs] link, but that's nitpicking :) It could just be [dictionary]

Reply:
done